core/services/keystore/keys/starkkey/export.go

```diff
@@ -0,0 +1,55 @@
+package starkkey
```

Comment:
added a generic function for the `FromEncryptedJSON` and `ToEncryptedJSON` functionality for all keys except for VRF and ETH keys since they use a different encryption format